packages/forma-36-react-components/src/components/Button/Button.tsx

```diff
@@ -51,11 +42,11 @@ export interface ButtonProps {
 
 export const Button = (props: ButtonProps) => {
   const {
```

Comment:
Do you want to move these to the function parameters now?

Reply:
Ohhhhh yes! Totally forgot about it

thanks